packages/upstream-extra/merlin.4.6-414/opam

```diff
@@ -0,0 +1,78 @@
+opam-version: "2.0"
+name: "merlin"
```

Comment:
I think merlin is vendored inside ocaml-lsp-server already (with some bugfixes on top), is it useful to keep this separate package, do we still use it directly?

Reply:
I was keeping it in case developers used it, if @lindig is okay with it, it can be dropped

Reply:
I am using merlin but for me it does not have to be in XS-opam